base/fabric8-analytics-data-model/Dockerfile

```diff
@@ -1,6 +0,0 @@
-FROM registry.access.redhat.com/rhel7
```

Comment:
please also remove this symlink https://github.com/rhdt/EL-Dockerfiles/blob/master/build_order/0-fabric8-analytics-data-model

Reply:
Removed now